cmd/next.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+var (
+	errNotSupportedPart = errors.New("part string is not a supported next version increment")
+)
+
+var nextCmd = &cobra.Command{
+	Use:   "next part",
+	Short: "Next increment version based on semantic versioning",
+	Long: `Next increment version based on semantic versioning.
+Check latest version and increment part (major, minor, patch).
+Echo the next release version number to be used by CI tools or other commands like batch.`,
+	Args: cobra.ExactArgs(1),
+	RunE: runNext,
+}
+
+func init() {
+	rootCmd.AddCommand(nextCmd)
+}
+
+func runNext(cmd *cobra.Command, args []string) error {
+	fs := afero.NewOsFs()
+	afs := afero.Afero{Fs: fs}
+
+	result, err := nextPipeline(afs, args[0])
```

Comment:
Can you Strings.ToLower the args[0] so "MAJOR" or "Major" will also work. No need for an error for capitals.

Reply:
solved in 649124d